src/AccordionSection/AccordionSection.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import { heightOfAccordionHeader } from "../Accordion/Accordion"
+import Icon from "../Icon/Icon"
+import { DefaultProps } from "../types"
+import { useUniqueId } from "../useUniqueId"
+import styled from "../utils/styled"
+
+export interface AccordionSectionProps extends DefaultProps {
+  title: React.ReactNode
+  children: React.ReactNode
+  id?: string
+  tabindex?: number
```

Comment:
```suggestion
  tabIndex?: number
```

From the [docs](https://reactjs.org/docs/dom-elements.html): ![image](https://user-images.githubusercontent.com/9947422/58461866-ed3b2380-8130-11e9-89be-c187025a6e45.png)